resources/js/components/forms/markdown.vue

```diff
@@ -3,63 +3,42 @@
-            <span class="caret"></span>
+          <button @click="makeBold" type="button" class="btn btn-sm"
+                  :title="state.canBold ? 'Dodaj pogrubienie' : 'Dodanie tutaj pogrubienia mogłoby uszkodzić składnię'"
+                  :style="{opacity: state.canBold ? '1.0' : '0.4', cursor: state.canStrikeThrough ? 'pointer' : 'default'}">
+            <i class="fas fa-bold fa-fw"></i>
           </button>
-
-          <vue-dropdown ref="dropdown" class="dropdown-menu select-menu">
-            <div class="select-menu-search">
-              <input v-model="searchText" @keyup.esc="hideDropdown" ref="search" type="text" class="form-control form-control-sm" placeholder="Filtruj..." autocomplete="off">
-            </div>
-
-            <div class="select-menu-wrapper">
-              <ul class="list-unstyled">
-                <li v-for="(value, key) in filteredLanguages">
-                  <a @click="insertAtCaret('```' + key + '<br>', '<br>```<br>')">{{ value }}</a>
-                </li>
-              </ul>
-            </div>
-          </vue-dropdown>
-        </div>
-
-        <div class="btn-group mr-2" role="group">
-          <button @click="insertAtCaret('<br>', '* ')" type="button" class="btn btn-sm" title="Wypunktowanie"><i class="fas fa-list-ul fa-fw"></i></button>
-          <button @click="insertAtCaret('<br>', '1. ')" type="button" class="btn btn-sm" title="Numerowanie"><i class="fas fa-list-ol fa-fw"></i></button>
-          <button @click="insertAtCaret('![', '](http://)')" type="button" class="btn btn-sm" title="Obraz"><i class="fas fa-image fa-fw"></i></button>
-          <button @click="insertAtCaret('[', '](http://)')" type="button" class="btn btn-sm" title="Link"><i class="fas fa-link fa-fw"></i></button>
-          <button @click="insertAtCaret('<br>Nagłówek 1 | Nagłówek 2<br>---------------- | -------------------<br>Kolumna1 | Kolumna 2<br>', '')" type="button" class="btn btn-sm" title="Tabela"><i class="fas fa-table fa-fw"></i></button>
+          <button @click="makeItalics" type="button" class="btn btn-sm"
+                  :title="state.canItalics ? 'Dodaj kursywę' : 'Dodanie tutaj kursywy mogłoby uszkodzić składnię'"
+                  :style="{opacity: state.canItalics ? '1.0' : '0.4', cursor: state.canStrikeThrough ? 'pointer' : 'default'}">
+            <i class="fas fa-italic fa-fw"></i>
+          </button>
+          <button @click="makeStrikeThrough" type="button" class="btn btn-sm"
+                  :title="state.canStrikeThrough ? 'Dodaj przekreślenie' : 'Dodanie tutaj przekreślenia mogłoby uszkodzić składnię'"
+                  :style="{opacity: state.canStrikeThrough ? '1.0' : '0.4', cursor: state.canStrikeThrough ? 'pointer' : 'default'}">
+            <i class="fas fa-strikethrough fa-fw"></i>
+          </button>
+          <label style="color:grey; align-self:center; margin: 3px 0 0;" title='"Smart paste" wkleja linki jako markdown'>
```

Comment:
Przeniósłbym te inline style, to `sass/components/forms/_markdown.scss`. Ogolnie docelowo dobrze by bylo wywalic komponent `markdown.vue` i zmienic nazwe tego Twojego na `markdown`